src/main/java/baseball/Application.java

```diff
@@ -1,7 +1,80 @@
 package baseball;
+import inputnumber.InputNumber;
+import makenumber.MakeNumber;
+import printcount.PrintCount;
+import camp.nextstep.edu.missionutils.*;
+import numbercompare.Compare;
 
+
+import java.util.ArrayList;
+import java.util.List;
+//
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        System.out.println("숫자 야구 게임을 시작합니다.");
+
+        List<Integer> computer = new ArrayList<>();
+        while (computer.size() < 3) {
+            int randomNumber = Randoms.pickNumberInRange(1, 9);
+            if (!computer.contains(randomNumber)) {
+                computer.add(randomNumber);
+            }
+        }
+        //System.out.println(computer + "컴퓨터 리스트");
+        boolean check = true;
+        while (check) {
+
+            System.out.print("숫자를 입력해주세요 : ");
+
+            // 이용자의 입력값
+            InputNumber inputnumber = new InputNumber();
+            String input = Console.readLine();
+            // 입력값 확인
+            inputnumber.NumberCheck(input);
+
+            // 볼, 스트라이크 판정
+            int [] k = Validation(input, computer);
+
+            int ball = k[0];
+            int strike = k[1];
+
+            PrintCount count = new PrintCount();
+            count.CountResult(ball,strike);
+
+            // 모두 맞춘 경우
+            if (k[1] == 3) {
+
+                System.out.println("3개의 숫자를 모두 맞히셨습니다! 게임 종료");
+                System.out.println("게임을 새로 시작하려면 1, 종료하려면 2를 입력하세요.");
+                int last = Integer.parseInt(Console.readLine());
+
+                if (last == 2) {
+                    check = false;
+                } else {
+                    MakeNumber number = new MakeNumber();
+                    number.numberList();
```

Comment:
한 게임을 마치고 재시작할 때 `computer` 변수에 새로운 랜덤값이 들어오지 않고 있습니다!
이 메소드에서 랜덤값을 받아올 때, `computer` 변수에 대입하면 될 것 같아요~

```suggestion
                    computer = number.numberList();
```

Reply:
아하 변수에 다시 담아야 하는건데.. 그냥 지나쳤군요.. ㅎㅎ 리뷰 감사합니다!